events_task/test/my_app_web/controllers/event_controller_test.exs

```diff
@@ -0,0 +1,224 @@
+defmodule MyApp.EventControllerTest do
+  use MyAppWeb.ConnCase
+
+  alias MyApp.Accounts
+  alias MyAppWeb.Guardian
+  alias MyApp.Content
+
+  @valid_attrs %{
+    "description" => "TestText",
+    "timestamp" => %{
+      "day" => "1",
+      "hour" => "0",
+      "minute" => "0",
+      "month" => "1",
+      "year" => "2015"
+    }
+  }
+  @invalid_attrs %{
+    "description" => "",
+    "timestamp" => %{
+      "day" => "1",
+      "hour" => "0",
+      "minute" => "0",
+      "month" => "1",
+      "year" => "2015"
+    }
+  }
+  describe "events" do
+    def create_test_user(attrs \\ %{}) do
+      {:ok, user} =
+        attrs
+        |> Enum.into(%{email: "sergei@gmail.com"})
+        |> Accounts.create_user()
+
+      user
+    end
+
+    def create_test_event(attrs \\ %{}) do
+      {:ok, event} =
+        attrs
+        |> Enum.into(@valid_attrs)
+        |> Content.create_event()
+
+      event
+    end
+
+    test "successfully open new event page" do
+      conn = get(build_conn(), event_path(conn, :new))
+      assert conn.path_info == ["events", "new"]
+    end
+
+    test "successfully event creation with valid params" do
+      conn =
+        build_conn()
+        |> Guardian.Plug.sign_in(create_test_user())
+        |> post(event_path(conn, :create, %{"event" => @valid_attrs}))
+
+      assert redirected_to(conn) =~ "/events/"
+    end
+
+    test "fails event creation with invalid params" do
+      conn =
+        build_conn()
+        |> Guardian.Plug.sign_in(create_test_user())
+        |> post(
+          event_path(conn, :create, %{
+            "event" => @invalid_attrs
+          })
+        )
+
+      assert response(conn, 200) =~ "Oops, something went wrong! Please check the errors below."
+    end
+
+    test "successfully show event" do
+      event = create_test_event()
+
+      conn =
+        build_conn()
+        |> Guardian.Plug.sign_in(create_test_user())
+        |> get(event_path(conn, :show, event.id))
+
+      assert response(conn, 200) =~ "<strong>Description:</strong>\n" <> event.description
+    end
+
+    conn
+
+    test "fails edit event with invalid user" do
+      conn = build_conn() |> Guardian.Plug.sign_in(create_test_user())
+
+      id =
+        conn
+        |> post(event_path(conn, :create, %{"event" => @valid_attrs}))
+        |> redirected_to()
+        |> String.slice(8..-1)
```

Comment:
This is unreliable way to retrieve an id - path to event show page can be changed anytime, and this test will break.
Since you are not checking `:create` action here - why not just create an event using `create_test_event`?